sdk/javascript/messaging/stream.js

```diff
@@ -94,7 +94,6 @@ class Stream {
   }
 
   send (type, content) {
-    console.log('Sending', Message.MessageType.stringValue(type))
```

Comment:
I mentioned it to him a few weeks ago and (IIRC) he said it was fine. The statements are low-level and very annoying. Here's an example:

```
integration_test_1      | INFO:test_intkey_smoke:Posting batch
rest_api_1              | [02:49:00.389 INFO     rest_api] Request  97fcf5: "POST /batches" from 172.18.0.6
rest_api_1              | [02:49:00.403 INFO     rest_api] Response 97fcf5: "202" status, 181B size, in 0.014s
tp_intkey_javascript_1  | Received TP_PROCESS_REQUEST(5)
tp_intkey_javascript_1  | Sending TP_STATE_GET_REQUEST(7)
tp_intkey_javascript_1  | Received TP_STATE_GET_RESPONSE(8)
tp_intkey_javascript_1  | Sending TP_STATE_SET_REQUEST(9)
tp_intkey_javascript_1  | Received TP_STATE_SET_RESPONSE(10)
tp_intkey_javascript_1  | Verb: inc Name: lark Value: 1
tp_intkey_javascript_1  | Received TP_PROCESS_REQUEST(5)
tp_intkey_javascript_1  | Sending TP_STATE_GET_REQUEST(7)
tp_intkey_javascript_1  | Received TP_STATE_GET_RESPONSE(8)
tp_intkey_javascript_1  | Sending TP_STATE_SET_REQUEST(9)
tp_intkey_javascript_1  | Received TP_STATE_SET_RESPONSE(10)
```